packages/eas-cli/src/commandUtils/EasCommand.ts

```diff
@@ -197,6 +197,15 @@ export default abstract class EasCommand extends Command {
       cleanMessage += err?.graphQLErrors?.[0]?.extensions?.requestId
         ? `\nRequest ID: ${err.graphQLErrors[0].extensions.requestId}`
         : '';
+
+      if("UNAUTHORIZED_ERROR" === err?.graphQLErrors?.[0]?.extensions?.errorCode){
+        cleanMessage = "Your Expo account may not have the minimum permission to do this operation." +
+          'For more information visit: https://github.com/expo/expo/issues/22696';
+
+        Log.error(cleanMessage);
+        throw new Error(cleanMessage);
+      }
```

Comment:
```suggestion
      if (err?.graphQLErrors?.[0]?.extensions?.errorCode === "UNAUTHORIZED_ERROR") {
        cleanMessage = `You don't have the required permissions to perform this operation.\n\n${cleanMessage}`;
      }
```
What do you think about it? 😄